### ISSUE TITLE
Upgrade Command Line Not Working on Windows

### ISSUE THREAD

**Issue by mc-drizzle** [appwrite/appwrite#326]
When the upgrade command: `docker exec appwrite upgrade` is run in Windows CMD or Powershell, the error returned is 'No such container: appwrite'

Please share the correct Windows command to upgrade Appwrite to the latest version, thanks again 
![Upgrade CMD Error](https://user-images.githubusercontent.com/65621952/82448940-bb8e3b80-9aaa-11ea-87a4-3f69daaae6b3.PNG)

**Comment by eldadfux** [comment]
You should run this command from the path where your appwrite docker-compose.yml is located.

If that doesn't work try replacing 'appwrite' with the Appwrite conatiner ID.

You can see your container ID using the `docker ps` command.

Let me know if this solves the issue.

**Comment by SpajicM** [comment]
@mc-drizzle @eldadfux problem here is that container isn't named "appwrite" but "appwrite_appwrite_1" and upgrade is missing a / as #329 stated. 

But even after that, upgrade isn't working:

> PHP Notice:  Undefined variable: requset in /usr/share/nginx/html/app/tasks/upgrade.php on line 21
Upgrading project: console
Starting Upgrade
Fetched 0 (offset: 0 / limit: 30) documents from a total of 0
Fetched 0 projects...


![image](https://user-images.githubusercontent.com/21988220/82494019-38d8a100-9ae9-11ea-9583-1d9fb7c2ab6a.png)

Then I changed the "requset" variable to "request", that fixed the PHP Notice warning but upgrade still not working, "fetched 0 projects".

**Comment by mc-drizzle** [comment]
> You should run this command from the path where your appwrite docker-compose.yml is located.
> 
> If that doesn't work try replacing 'appwrite' with the Appwrite conatiner ID.
> 
> You can see your container ID using the `docker ps` command.
> 
> Let me know if this solves the issue.

My docker-compose.yaml file is located in the file 'appwrite' so I tried the command from that path, but still with no success, even when using the container ID, I also tried using the suggestion of @SpajicM but mine also fetched 0 projects
![Another Upgrade Failure](https://user-images.githubusercontent.com/65621952/82516399-e57f4680-9b1a-11ea-8bed-570bd91a4373.PNG)

**Comment by eldadfux** [comment]
This is OK if your running on a new setup with no previous data, we should probably add a better log message to let you know the operation was successful.

**Comment by mc-drizzle** [comment]
@eldadfux yes, I am running on a new setup with no previous data. 

So by 'OK' do you mean that the version should have updated to 0.6.0?

Because mine is still at 0.5.3
![Current Version](https://user-images.githubusercontent.com/65621952/82519989-5c204200-9b23-11ea-8086-6ae01c4575ed.PNG)

**Comment by eldadfux** [comment]
The upgrade command should be run after the setup command (i'll make sure to make this more clear in the docs).

Run this command from the parent directory to your appwrite directory where you have your docker-compose.yml file:

* parent <- **from this location**
    * appwrite <- **not from this**
        * docker-compose.yml
```
docker run -it --rm \
    --volume /var/run/docker.sock:/var/run/docker.sock \
    --volume "$(pwd)"/appwrite:/install/appwrite:rw \
    -e version=0.6.0 \
    appwrite/install
```

**Comment by mc-drizzle** [comment]
> The upgrade command should be run after the setup command (i'll make sure to make this more clear in the docs).
> 
> Run this command from the parent directory to your appwrite directory where you have your docker-compose.yml file:
> 
> * parent <- **from this location**
>   
>   * appwrite <- **not from this**
>     
>     * docker-compose.yml
> 
> ```
> docker run -it --rm \
>     --volume /var/run/docker.sock:/var/run/docker.sock \
>     --volume "$(pwd)"/appwrite:/install/appwrite:rw \
>     -e version=0.6.0 \
>     appwrite/install
> ```
@eldadfux is not that 
a command to install appwrite? I already have apprite installed (but version 0.5.3) and I did install it from the parent path.

So if I run the code you gave, will it create a **new** appwrite container or will it update my current version to 0.6.0?

Thanks again for your awesome assistance

**Comment by eldadfux** [comment]
If run from the parent directory of the original setup, it will overwrite your current docker-compose.yml and restart your stack. If run from a new location it will create a second Appwrite instance (as long as your ports aren't conflicting).

You can also manually update your Appwrite version in your docker-compose.yml, and restart your instance with `docker-compose up -d`

**Comment by mc-drizzle** [comment]
Okay thanks a lot! It did work...

I'm now running on version 0.6.0
![Updated Appwrite](https://user-images.githubusercontent.com/65621952/82809770-236aca80-9e8d-11ea-82dd-7f2bb2e7bd59.PNG)

I ran the code from the parent directory of the original setup, and it managed to overwrite, but when I check back to the console, it's showing an older version:
![Appwrite Console Showing Old Version](https://user-images.githubusercontent.com/65621952/82809938-89575200-9e8d-11ea-9571-fb0ba92e362e.PNG)
Is there an additional step I need to take to have the console running on the latest version? 

Thanks again!

**Comment by eldadfux** [comment]
Maybe just a cache issue? Also 0.6.1 is already out :)

**Comment by mc-drizzle** [comment]
Okay thanks, I cleared my cache and reloaded the console, it doesn't show the old version anymore but it says "**unknown**":
![Unknown Version](https://user-images.githubusercontent.com/65621952/82837964-f50be000-9eca-11ea-8647-53817e644404.PNG)
But as far as I can tell, I'm running on v0.6.0 as shown in the previous screenshot, and my Docker Dashboard shows v0.6.0 as well.

Thanks again! I really appreciate your help! And I'll be sure to update to v0.6.1!

**Comment by eldadfux** [comment]
Thats great!

The UNKNOWN version bug was already fixed in 0.6.1...

**Comment by mc-drizzle** [comment]
@eldadfux Your the BEST! I updated to v0.6.1, and we're all good here:
![V061](https://user-images.githubusercontent.com/65621952/82838792-93994080-9ecd-11ea-821a-fc8a82ae6022.PNG)

Your support has been **tremendous**, thanks again!
